mypy/errors.py

```diff
@@ -10,14 +10,19 @@
 from mypy.scope import Scope
 from mypy.options import Options
 from mypy.version import __version__ as mypy_version
-from mypy.errorcodes import ErrorCode
+from mypy.errorcodes import ErrorCode, IMPORT
 from mypy import errorcodes as codes
 from mypy.util import DEFAULT_SOURCE_OFFSET, is_typeshed_file
 
 T = TypeVar('T')
 allowed_duplicates = ['@overload', 'Got:', 'Expected:']  # type: Final
 
 
+# Threshold after which we may filter out most errors to avoid very
+# verbose output
+MANY_ERRORS_THRESHOLD = 200
```

Comment:
I think this should be tunable by user (e.g. with a flag). For example, what if the errors are caused by something else (not missing imports). I always prefer to see all the errors especially when working with the PyCharm plugin.

Finally, testing will be simpler when it is tunable.